lib/js/gbc.js

```diff
@@ -309,6 +382,9 @@ window.tsfGBC = function( $ ) {
 		subscribe( saveDispatcher );
 
 		document.dispatchEvent( new CustomEvent( 'tsf-subscribed-to-gutenberg' ) );
+		document.dispatchEvent(
+			new CustomEvent( 'tsf-gutenberg-tt', { detail: overrideTooltip } )
+		);
```

Comment:
Using an event for this just seemed nice to not add publicly accessible members though of course it’s still public 🤷.